internal/app/subsystems/aio/store/postgres/postgres.go

```diff
@@ -224,6 +242,40 @@ const (
 
 	NOTIFICATION_DELETE_STATEMENT = `
 	DELETE FROM notifications WHERE id = $1 AND promise_id = $2`
+
+	SCHEDULE_SELECT_STATEMENT = `
```

Comment:
Can you check the indentation here? Also can we order these commands
- promises
- schedules
- everything else